### PR TITLE
What: Improve readability/structure and content clarity

### PR DESCRIPTION
**Pull request by TheFoxAtWork**
Why:

* initial draft is excellent but additional clarity and consistency improves readability and contribution potential

This change addresses the need by:

* adjusting various areas for context-aligned nesting
* add clarifying information for some content areas
* apply consistent header/content spacing & newline

### CODE REVIEW

**Comment by catblade** [comment]
@TheFoxAtWork  2 questions:
1)  I merged this with the current PR.  Can you double-check this is correct?  (Two PRs on a working doc means headaches for all).  The only thing I did out-of-band of merging was to remove the extraneous scheduling section (there is one at the bottom of the page) and to fix a couple typos.  Oh, I guess I also added v.0.1, as we discussed in the community chat earlier today.
2) Do you mind if I add your name to the contributors?  I'm trying to make sure everyone who does contribute gets credit for that work.  (Leo and Max still need to add theirs, but it'll get done).  This has been a huge lift, and I'd like to make sure I mark who has helped.

File: documentation/WhitePapers/SustainabilityUseCasesAndLandscape2022.md [hunk 1]
```diff
@@ -1,67 +1,88 @@
...
+# Current Sustainability Efforts and Challenges Within the Cloud Native Landscape V. 0.1
+
+This captures the known and ongoing sustainbility efforts within the cloud native landscape as well as identifies challenge areas.
 
 ## Contributors
+A special thank you to our contributors of this document. If you are interested in improving and enhancing the content, please file a PR on the repo and ensure you add yourself as a contributor below!
+
 Huamin Chen, [Marlow Weston](https://github.com/catblade), [Niki Manoledaki](https://github.com/nikimanoledaki), Eun Kyung Lee, Chen Wang, Chris Lloyd-Jones, 
-[Parul Singh](https://github.com/husky-parul), [Przemysław Perycz](https://github.com/pperycz), [Christopher Cantalupo](https://github.com/cmcantalupo), [Paricia Cahill](https://github.com/patricia-cahill), [Jochen Joswig](https://github.com/by-d-sign)
+[Parul Singh](https://github.com/husky-parul), [Przemysław Perycz](https://github.com/pperycz), [Christopher Cantalupo](https://github.com/cmcantalupo), [Patricia Cahill](https://github.com/patricia-cahill), [Jochen Joswig](https://github.com/by-d-sign)
```
Comment (TheFoxAtWork) [inline documentation/WhitePapers/SustainabilityUseCasesAndLandscape2022.md:9]:
```suggestion
[Parul Singh](https://github.com/husky-parul), [Przemysław Perycz](https://github.com/pperycz), [Christopher Cantalupo](https://github.com/cmcantalupo), [Patricia Cahill](https://github.com/patricia-cahill), [Jochen Joswig](https://github.com/by-d-sign), [Emily Fox](https://github.com/thefoxatwork)
```

**Review by TheFoxAtWork — COMMENTED** [review]
Looks like everything merged in correctly.  I've suggested the change to list me as a contributor (apologies i just finished packing everything up for KCCN)

**Comment by catblade** [comment]
Merging, and many thanks!

**Pull request merged**